pkg/utils/utils.go

```diff
@@ -26,18 +26,12 @@ func GetBytes(key interface{}) ([]byte, error) {
 }
 
 // ParseURL extracts database parameters from a string as a URL
-//   bolt://storj.db
-//   bolt://C:\storj.db
-//   redis://hostname
-func ParseURL(s string) (*url.URL, error) {
-	if strings.HasPrefix(s, "bolt://") {
-		return &url.URL{
-			Scheme: "bolt",
-			Path:   strings.TrimPrefix(s, "bolt://"),
-		}, nil
+func ParseURL(s string) (string, string, error) {
```

Comment:
Don't remove the examples these are necessary to understand why it's not using `url.URL` directly.

Maybe let's rename this to `ParseDatabaseURI` or `ParseDatabaseURL` something like that.